File: Api.Server/src/Services/Center/File/Dto/QueryFilePagedInput.cs
namespace Fast.Center.Service.File.Dto;

/// <summary>
/// <see cref="QueryFilePagedInput"/> 获取文件分页列表输入
/// </summary>
public class QueryFilePagedInput : PagedInput
{
    /// <summary>
    /// 租户Id
    /// </summary>
    public long? TenantId { get; set; }
}

File: App.Admin/src/api/services/file/models/QueryFilePagedInput.ts
/**
 * Fast.Center.Service.File.Dto.QueryFilePagedInput 获取文件分页列表输入
 */
export interface QueryFilePagedInput extends PagedInput  {
  /**
   * 租户Id
   */
  tenantId?: number;
  /**
   * 
   */
  readonly isOrderBy?: boolean;
}

